src/util.rs

```diff
@@ -22,9 +22,19 @@ pub fn path_to_bytes<P: AsRef<Path>>(path: &P) -> Option<&[u8]> {
 }
 
 pub fn get_db() -> Result<DB> {
-    let mut path = config::zo_data()?;
-    path.push("db.zo");
-    DB::open(path)
+    let mut db_file = config::zo_data_dir()?;
```

Comment:
The variable name here is misleading, since this is not a file. How about something like `db_path`?

Reply:
Not a file *yet*... Point taken; done.